pkg/scheduler/core/generic_scheduler.go

```diff
@@ -677,6 +677,9 @@ func PrioritizeNodes(
 	nodes []*v1.Node,
 	extenders []algorithm.SchedulerExtender,
 ) (schedulerapi.HostPriorityList, error) {
+	trace := utiltrace.New(fmt.Sprintf("Ranking nodes for %s/%s", pod.Namespace, pod.Name))
```

Comment:
These debugging calls were imposed in the critical path of every scheduling cycle. My concerns are:

1. How to evaluate the perf impact?
1. Is there a better way to make it configurable so that it can be enabled by users?

cc @k82cn @bsalamat @ravisantoshgudimetla for thoughts.

Reply:
While tracing is not really expensive (each step just reads the time, which is a sub-millisecond operation), I think for this case tracing may not be very useful since the function takes less than 50ms, and so profiling might be a better tool here.